lib/api/sentry_api.dart

```diff
@@ -20,7 +20,11 @@ class SentryApi {
 
   final String? authToken;
 
-  final client = sentry.SentryHttpClient(client: Client());
+  final client = sentry.SentryHttpClient(
+    captureFailedRequests: true,
+    sendDefaultPii: false,
+    failedRequestStatusCodes: [sentry.SentryStatusCode.range(500, 599)],
```

Comment:
I'm not quite sure if this is the best setting. Probably 4xx are more interesting, because they're potentially the apps fault?

Reply:
```suggestion
    failedRequestStatusCodes: [sentry.SentryStatusCode.range(400, 599)],
```

Reply:
yep 4xx 5xx makes sense